app/components/containerInspect/containerInspect.html

```diff
@@ -0,0 +1,19 @@
+<rd-header>
+  <rd-header-title title="Container inspect">
+  </rd-header-title>
+  <rd-header-content>
+    <a ui-sref="containers">Containers</a> &gt; <a ui-sref="container({id: containerInfo.Id})">{{ ::containerInfo.Name|trimcontainername }}</a> &gt; Inspect
```

Comment:
Please replace `::containerInfo.Name|trimcontainername` with `containerInfo.Name|trimcontainername` otherwise the link will not be shown.